.pre-commit-config.yaml

```diff
@@ -1,11 +1,34 @@
 repos:
+  - repo: local
+    hooks:
+      - id: avoid-old-names
+        name: Avoid old names and use use "Welkin" instead
+        entry: "(Compliant[ ]Kubernetes|C[kK]8[sS][^_]|Complaint)"
```

Comment:
Nit;
```suggestion
        entry: "(Compliant[ ]Kubernetes|C[kK]8[sS][^_]|Compliant)"
```

Reply:
This is based on @cristiklein additions to Apps, so was the intention to catch previous misspellings (as I assumed) or is this just a misspelling? :sweat_smile:

Else I need to go round and do some updating, and I don't know how well it would work as the word Compliant itself shouldn't be banned right?

Reply:
Yeah right I would assume it's supposed to catch the misspelling, my bad!

Reply:
@aarnq and @OlleLarsson I noticed a few usages of the misspelled "Complaint Kubernetes" when I migrated the public docs to Welkin. So, I made sure that the pre-commit hook issues complaints (pun intended) about those too. :smile: